extensions/wikia/UserRenameTool/process/ProcessBase.class.php

```diff
@@ -0,0 +1,556 @@
+<?php
+
+namespace UserRenameTool\Process;
+
+use Wikia\Logger\Loggable;
+use Wikia\Logger\WikiaLogger;
+
+class ProcessBase {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "ProcessBase" has 30 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)